src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -7454,6 +7547,30 @@ .maxstack  1
 ");
         }
 
+        [Fact]
+        public void UnmanagedOnUnsupportedRuntime()
+        {
+            var comp = CreateCompilationWithFunctionPointers(@"
+#pragma warning disable CS0168 // Unused variable
+class C
+{
+    unsafe void M()
+    {
+        delegate* unmanaged<void> ptr1;
+        delegate* unmanaged[Stdcall, Thiscall]<void> ptr2;
```

Comment:
>delegate* unmanaged[Stdcall, Thiscall]<void> ptr2; [](start = 8, length = 50)

I think we want to verify the shape of the symbol for scenario when runtime doesn't support the feature.